scripts/count_gene_mutations.R

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env Rscript
```

Comment:
May be worth pointing out (to emphasize further after scripting) that this path is not strictly universal, but also one doesn't have to rely on it.

Reply:
I expect to mention this in slides, yes. (This one is pretty universal though!)

Reply:
Mine's in `/usr/local/bin/`, is the `env` from a conda installation or similar? Just curious!

Reply:
That is odd! Do you maybe have it in both places and `local` is first in your PATH? As far as conda, the version installed with conda will be somewhere else, but the way `env` works it should still launch commands from the _current_ environment, regardless of which environment the `env` program resides in. For example, when I run `/usr/bin/env python` I get whatever version of python is active in my conda environment, not the base python (same as if I ran `env python`

Reply:
If you thought that was odd, wait til you read that... I do not have a `/usr/bin/env/` directory at all 
<img width="467" alt="Screen Shot 2022-05-23 at 4 48 38 PM (2)" src="https://user-images.githubusercontent.com/4701111/169903671-104c051b-f71b-4e4b-95ab-10808beca163.png">

Reply:
`/usr/bin` is the directory, which I really hope you have! `env` should be inside it, but is a binary (or maybe link)

Reply:
Lord help me I absolutely have been reading this _without a space_ as `usr/bin/env/Rscript`. 
I have it all, it just loses to `/usr/local/bin` in my `$PATH`